job.nomad.hcl

```
@@ -0,0 +1,49 @@
+job "ddagen" {
+  type = "service"
+
+  group "ddagen" {
+    network {
+      port "http" { }
+    }
+
+    service {
+      name     = "ddagen"
+      port     = "http"
+      provider = "nomad"
+      tags = [
+        "traefik.enable=true",
+        "traefik.http.routers.ddagen.rule=Host(`ddagen.se`)",
```

```suggestion
        "traefik.http.routers.ddagen.rule=Host(`ddagen.se`)||Host(`www.ddagen.se`)",
```
boomers like to add `www.` before addresses for fun